src/styles/utils/reser.scss

```diff
@@ -0,0 +1,5 @@
+body,
```

Comment:
Typo in the file name, it should be 'reset.scss' instead of 'reser.scss'. Please rename the file.